src/_gettsim/taxes/zu_verst_eink/vorsorgeaufw.py

```diff
@@ -2,12 +2,12 @@
 
 
 @dates_active(
-    start="2005-01-01", end="2022-12-31", change_name="vorsorgeaufw_alter_y_tu"
+    start="2005-01-01", end="2022-12-31", change_name="vorsorgeaufw_alter_y_sn"
 )
-def vorsorgeaufw_alter_y_tu_einfuehrung(
-    ges_rentenv_beitr_m_tu: float,
-    priv_rentenv_beitr_m_tu: float,
-    anz_erwachsene_tu: int,
+def vorsorgeaufw_alter_y_sn_einfuehrung(
+    ges_rentenv_beitr_m_sn: float,
```

Comment:
Do the conversion outside of the function. Applies to lots of things in this module, it seems by glancing at the top. 

But feel free to open an issue for that, I thought we had converted all that at some point already.

Reply:
I'll create an issue because this would entail rewriting some of the functions /
creating intermediate functions.

Reply:
Most of the functions in this module only make sense on the aggregated level because
they involve Aufwendungen that are capped at the `sn` level. Of course,
we could split up these functions into a function that calculates Aufwendungen on the
individual level and a part that caps them at the `sn` level. But it seems like we
wouldn't gain much clarity in this module because the functions for which this would
work are rather short (e.g. the one you highlighted here).

Reply:
Sorry, I meant the conversion from monthly to annual!

Reply:
Makes sense! Created #709